ui/nuxeo-page-provider-display-behavior.js

```diff
@@ -780,7 +807,13 @@ export const PageProviderDisplayBehavior = [
     _updateSelectedItems(selectionCallback) {
       const { start, end } = this._getSelectionBoundaries();
       for (let index = start; index <= end; index++) {
-        selectionCallback(index);
+        if (
+          !this._excludedItems ||
+          !this._excludedItems.length ||
+          (this.items[index] && !this._excludedItems.includes(this.items[index].uid))
+        ) {
+          selectionCallback(index);
+        }
```

Comment:
This prevents the selection callback from being called when and item is excluded, which prevents it from toggling the selection. Reverting this seems to have solved one of the selection issues with the grid for me locally.